src/blockchain.js

```diff
@@ -125,8 +138,12 @@ let Contract = function (options) {
 
   if (Contract.isNewWeb3(this.web3)) {
```

Comment:
I was going to ask, do we still support web3 0.x. I feel we removed it from embark but not embarkjs.

If not then all this code goes away.
Also I would have to check with old web3 and new metamask the behaviour, I doubt it will go well :)

Reply:
I'm down to remove the old web3. @iurimatias, you down?

Reply:
we don't support it anymore, can be removed

Reply:
I will add a second PR for the cleaning